app/views/users/show.html.erb

```diff
@@ -1 +1,38 @@
-<h1>Here is a page dedicated to a given user</h1>
\ No newline at end of file
+
+<div class='user-index'>
+    <div class='colored-a'>
+        <div class='user-div'>
+            <div>
+              <img src="<%= @user.photo %>">
+            </div>
+            <div class='name-and-post'>
+                <p><%= @user.name %><p>
+                <p>Number of posts: <%= @user.posts.count %><p>
+            </div>
+        </div>
+    </div>
+</div>
+
+<div class='bio'>
+    <span>Bio:</span>
+    <p><%= @user.bio %></p>
+</div>
```

Comment:
## [OPTIONAL]
- [ ] As you have already a partial defined in the above review comment, you will not need again to add this code above just call the partial `app/views/users/_user.html.erb`.

```html.erb
    <%= render @user %>